sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp

```diff
@@ -29,54 +29,153 @@ namespace Azure { namespace Storage {
 
     namespace Models {
 
+      /**
+       * @brief Response type for #Azure::Storage::Blobs::BlobClient::DownloadTo.
+       */
       struct DownloadBlobToResult
       {
+        /**
+         * The blob's type.
+         */
         Models::BlobType BlobType;
+
+        /**
+         * Indicates the range of bytes returned.
```

Comment:
```suggestion
         * Indicates the #Azure::Core::Http::HttpRange of bytes returned.
```

Reply:
The docs will look like

![image](https://user-images.githubusercontent.com/52266153/116191165-6f3d8800-a75e-11eb-8dde-8a4dde46df6a.png)
![image](https://user-images.githubusercontent.com/52266153/116191178-75336900-a75e-11eb-8f4a-a3af8c6c0c58.png)

So it shows the type very clearly. I don't feel it's necessary to repeat it.